oauth2_http/java/com/google/auth/http/AuthHttpConstants.java

```diff
@@ -0,0 +1,16 @@
+package com.google.auth.http;
```

Comment:
Hmm, I opened up other files to copy their license header, and it looks like it's missing from most of the files. Under oauth2_http in particular, only ServiceAccountJwtAccessCredentials.java has it. Question: Do you want me to add the license header to just this file for this change? Or, not add it, but do a follow-up change adding the license header to all of the files?